server/pyproject.toml

```diff
@@ -81,5 +81,5 @@ rabbit_user_queues = "rabbitmq_user_queues:GirderPlugin"
 dive_tasks = "dive_tasks:DIVEPlugin"
 
 [build-system]
-requires = ["poetry-core>=1.0.0"]
+requires = ["poetry-core>=1.0.0", "setuptools"]
```

Comment:
Would it be better to specify a [compatible version](https://www.python.org/dev/peps/pep-0440/#compatible-release)?

Reply:
All future versions of setuptools should be compatible. I don't know how I'd restrict this version, because I don't want to use an outdated version or prevent upgrade in the future.